example/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,6 @@
 <manifest
     xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="com.example.clevertap_plugin_example">
```

Comment:
Removal of package needs to be tested for compatibility with lower versions of flutter. Will verify once on my end